lib/collectinfo_analyzer/show_controller.py

```diff
@@ -916,17 +1051,204 @@ def do_sindex(self, line):
+    )
+    def do_dc(self, line):
+        show_total = util.check_arg_and_delete_from_mods(
+            line=line, arg="-t", default=False, modifiers=self.modifiers, mods=self.mods
+        )
+        title_every_nth = util.get_arg_and_delete_from_mods(
+            line=line,
+            arg="-r",
+            return_type=int,
+            default=0,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        )
+
+        flip_output = util.check_arg_and_delete_from_mods(
+            line=line,
+            arg="-flip",
+            default=False,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        ) or util.check_arg_and_delete_from_mods(
+            line=line,
+            arg="--flip",
+            default=False,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        )
+
+        xdr_dc_stats = self.getter.get_xdr_dcs(for_mods=self.mods["for"])
+
+        for timestamp in xdr_dc_stats.keys():
+            cinfo_log = self.log_handler.get_cinfo_log_at(timestamp=timestamp)
+
+            self.view.show_xdr_dc_stats(
+                xdr_dc_stats[timestamp],
+                cinfo_log,
+                title_every_nth=title_every_nth,
+                flip_output=flip_output,
+                show_total=show_total,
+                timestamp=timestamp,
+                **self.mods,
+            )
+
+    @CommandHelp(
+        "Displays xdr namespace statistics.",
+        "  Options:",
+        "    -t           - Set to show total column at the end. It contains node wise sum for statistics.",
+        "    -r           - Repeat output table title and row header after every <terminal width> columns.",
+        "                   [default: False, no repetition]",
+        "    --flip       - Flip output table to show Nodes on Y axis and stats on X axis.",
+    )
+    def do_namespace(self, line):
+        show_total = util.check_arg_and_delete_from_mods(
+            line=line, arg="-t", default=False, modifiers=self.modifiers, mods=self.mods
+        )
+        title_every_nth = util.get_arg_and_delete_from_mods(
+            line=line,
+            arg="-r",
+            return_type=int,
+            default=0,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        )
+
+        flip_output = util.check_arg_and_delete_from_mods(
+            line=line,
+            arg="-flip",
+            default=False,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        ) or util.check_arg_and_delete_from_mods(
+            line=line,
+            arg="--flip",
+            default=False,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        )
+
+        by_dc = util.check_arg_and_delete_from_mods(
+            line=line,
+            arg="--by-dc",
+            default=False,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        )
```

Comment:
Does the `--by-dc` modifier need help text?

Reply:
yes, good catch